tick-api/src/main/java/me/aurium/tick/ImageManager.java

```diff
@@ -0,0 +1,9 @@
+package me.aurium.tick;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface ImageManager {
+
+    CompletableFuture<String> loadImageID();
```

Comment:
Don't make return types stringly-typed

Reply:
The image ID has to be an object in the docker api